roles/provision-metrics-apb/tasks/provision-app-metrics.yml

```diff
@@ -0,0 +1,115 @@
+- name: Postgresql PVC
+  k8s_v1_persistent_volume_claim:
+    name: postgresql-data
+    namespace: "{{ namespace }}"
+    labels:
+      app: app-metrics-api
+      service: prometheus
+    access_modes:
+      - ReadWriteMany
+    resources_requests:
+      storage: "10Gi"
+    state: present
+
+- name: create postgres deployment config
+  openshift_v1_deployment_config:
+    name: postgres
+    namespace: '{{ namespace }}'
+    labels:
+      app: postgres
+      service: prometheus
```

Comment:
@david-martin @philbrookes I continued to use prometheus as a service label everywhere as that is whats being done in the APB but should/could we remove or change this? maybe to metrics?

Reply:
See comment above about service label.
This has a specific meaning, so lets be careful we don't wire up the wrong service to pods.
See https://kubernetes.io/docs/concepts/services-networking/service/
`The set of Pods targeted by a Service is (usually) determined by a Label Selector`